doc/mkdocs/docs/api/macros/json_has_ranges.md

```diff
@@ -0,0 +1,23 @@
+# JSON_HAS_RANGES
+
+```teratermmacro
+#define JSON_HAS_RANGES /* value */
+```
+
+Wether the standard library has any support for ranges. Implies support for concepts.
+Possible values are `1` when supported or `0` when unsupported.
+
+## Default definition
+
+Unless defined by the user prior to inclusion of the `json.hpp` header, the macro definition is roughly equivalent to the follwing:
+```teratermmacro
```

Comment:
Whoops. I was experimenting with syntax highlighting. Those should all be `cpp`. Sadly the highlighting with `pygments` is quite monotone.